cpp/include/Ice/Format.h

```diff
@@ -7,12 +7,14 @@
 
 namespace Ice
 {
-    /// Describes the possible formats for classes and exceptions.
+    /// Specifies how to marshal classes and exceptions with the Ice 1.1 encoding.
```

Comment:
What about `Specifies the format for marshaling classes...`

Reply:
Fixed as suggested.